device/dell/x86_64-dellemc_z9332f_d1508-r0/plugins/psuutil.py

```diff
@@ -46,28 +42,25 @@ def isDockerEnv(self):
 
     # Fetch a BMC register
     def get_pmc_register(self, reg_name):
-
         status = 1
```

Comment:
>status = 1 [](http://example.com/codeflow?start=8&length=10)

lgtm said "This assignment to 'status' is unnecessary as it is redefined here before this value is used."